src/components/transactions/TransactionItem.vue

```diff
@@ -73,7 +74,9 @@ export default {
     }
   },
   data: () => ({
-    show: false
+    show: false,
```

Comment:
do we still need this is `show-validators`?

Reply:
no, this one is the `show` for the transaction metadata, so yes, we still need it here

Reply:
so this should be called `showMetaData` probably 😸

Reply:
wait, there is already a prop called `showMetaData` on the `TransactionMetaData` component.